django/db/models/base.py

```diff
@@ -900,7 +902,7 @@ def save_base(
 
     save_base.alters_data = True
 
-    def _save_parents(self, cls, using, update_fields):
+    def _save_parents(self, cls, using, update_fields, force_insert=False):
```

Comment:
My preference would be that we don't put a default for `force_insert` as it's a private method only called in one location outside any recursive calls.